joomla/joomla-cms
# Update color form field to support opacity with minicolor

Update color form field type use to latest jQuery minicolor. Adds support for rgba colors with new attributes `format` and `opacity`.
## How to test

Create two color fields and check if everything works like expected.

**Expected:** results in `hex` color and hasn't opacity picker.

```
<field
    name="colorhex"
    type="color" />
```

**Expected:** results in `rgb` color and hasn't opacity picker.

```
<field
    name="colorrgba"
    type="color" 
    format="rgb" />
```

**Expected:** results in `rgba` color and has opacity picker.

```
<field
    name="colorrgba"
    type="color" 
    format="rgb"
    opacity="0.5" />
```
